database/src/pool/postgres.rs

```diff
@@ -1349,3 +1358,15 @@ fn parse_artifact_id(ty: &str, sha: &str, date: Option<DateTime<Utc>>) -> Artifa
         _ => panic!("unknown artifact type: {:?}", ty),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::make_certificates;
+
+    // Makes sure we successfully parse the RDS certificates and load them into native-tls compatible
+    // format.
+    #[test]
+    fn can_make_certificates() {
+        make_certificates();
```

Comment:
I know that you took this from triagebot, but can we also just do a small assert that the resulting certificate chain is not empty? Thanks!

Reply:
Sure! Done 👍